Unstructured-IO/unstructured
# docs: small tweak, remove metadata row

**Test Instructions**

    cd docs
    pip install -r requirements.txt
    make html

Verify the metadata page is not broken and the non-applicable row for `num_characters` is removed.
